pandas/io/parsers.py

```diff
@@ -2153,8 +2155,12 @@ def _try_convert_dates(parser, colspec, data_dict, columns):
 
     new_name = '_'.join([str(x) for x in colnames])
     to_parse = [data_dict[c] for c in colnames if c in data_dict]
-
     new_col = parser(*to_parse)
+    if hasattr(new_col, 'dtype') and any(r in str(new_col.dtype) for r in
```

Comment:
rebase on master, and just delete them, see http://pandas-docs.github.io/pandas-docs-travis/contributing.html#getting-started-with-git

you basically

``git rebase -i master``